README.md

```diff
@@ -10,9 +10,23 @@ This version is rewritten to use Python twisted, so this module is required.
 To start at run time:
 echo "/usr/bin/pyclustercheck -f /etc/my.cnf > /var/log/messages 2>&1 &" >> /etc/rc.local
```

Comment:
No related to this PR, just notice this line is dangerous and would wipe out /var/log/messages on run; this will need correction.

Reply:
pyclustercheck is also not the script name :)